src/UI/Denominate/index.tsx

```diff
@@ -1,11 +1,7 @@
 import React from 'react';
 import classNames from 'classnames';
 import globalStyles from 'assets/sass/main.scss';
-import {
-  decimals as configDecimals,
-  denomination as configDenomination,
-  mainnetEgldLabel
-} from 'constants/index';
+import { DECIMALS, DENOMINATION, mainnetEgldLabel } from 'constants/index';
```

Comment:
if we make an exception for  `mainnetEgldLabel`, we should also keep `decimals` and `denomination`

Reply:
It's not an exception, it's a miss